promotion/avro.go

```diff
@@ -51,3 +51,22 @@ type AdminAttestationEvent struct {
 	Justification string `json:"justification"`
 	CreatedAt     string `json:"created_at"`
 }
+
+const grantRewardsEventSchema = `{
+	"type": "record",
+	"name": "grantRewards",
+	"doc": "This message is from when a client claimed a swap reward",
+	"fields": [
+		{ "name": "promotion_id", "type": "string" },
+		{ "name": "address_id", "type": "string" },
+		{ "name": "transaction_key", "type": "string" },
+		{ "name": "reward_amount", "type": "string" }
```

Comment:
Would an integer type be better here? Either way, I think we'll probably want to represent the values in the equivalent of satoshis for BAT to avoid precision issues.  I think there is probably a precedent for passing BAT values in the that we could look at.

Reply:
Precision issues make an integer not good for this.